### PR TITLE
Update tech-specs.yml

### PR DESCRIPTION
**Pull request by akash73**
added MAx input voltage value

## What This PR Changes
- added MAx input voltage value to 10V

## Contribution Guidelines
- [X] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/07.opta/opta-family/opta/tech-specs.yml
Comment (jcarolinares) [inline content/hardware/07.opta/opta-family/opta/tech-specs.yml:11]:
```suggestion
  Max input Voltage: 10 V
```

It's a nice addition.

Changed to separate the 10V to 10 V to keep consistency

**Review by jcarolinares — CHANGES_REQUESTED** [review]

**Comment by akash73** [comment]
@jcarolinares please do not merge now this, waiting a final decision , because it seems that the Max is 24V . 
thanks

**Pull request closed without merging**
